docs/reference/migration/migrate_5_0/search.asciidoc

```diff
@@ -128,6 +128,7 @@ in favour of `query` and `no_match_query`.
 
 * Deprecated support for the coerce, normalize, ignore_malformed parameters in GeoPolygonQuery. Use parameter validation_method instead.
 
+* Deprecated support for the coerce, normalize, ignore_malformed parameters in GeoDistanceRangeQuery. Use parameter validation_method instead.
```

Comment:
> There only is a link to https://www.elastic.co/guide/en/elasticsearch/reference/master/query-dsl-geo-distance-range-query.html which has ignore_malformed explained, but not coerce.

This page explains `ignore_unmapped`, not `ignore_malformed`.  I was under the impression that `ignore_malformed` and  `coerce` were only used in mappings, not in queries?

> I see it being done e.g. here https://www.elastic.co/guide/en/elasticsearch/reference/2.3/mapping-transform.html with some warning box, should we show something similar for query parameters, or is it enough to list those in the migration doc? 

In the docs for the version in which it is deprecated, add a deprecated annotation, eg 

    deprecated[2.3.0,Use `foo` instead]

This annotation can be on its own line (rendered as a block) or inline (rendered inline)

> Related to that, how do we deal with formerly valid but undocumented query parameters?

Ignore them.